site/en/docs/devtools/javascript/background-services/index.md

```diff
@@ -170,6 +170,8 @@ To view the reports generated by a page:
 
 1. Click on a report to see its body.
 
+Additionally, the **Reporting API** gives you an overview of all the endpoints configured in the `Reporting-Endpoints` header.
```

Comment:
Probably it should be a pointer too. Or, we should expand the last point "Click on a report to see its body" to explain the 3 sections of the report